coalib/settings/ConfigurationGathering.py

```diff
@@ -201,12 +210,13 @@ def warn_config_absent(sections, argument, log_printer=None):
         argument = [argument]
     for section in sections.values():
         if any(arg in section for arg in argument):
-            return False
+            return (False, argument)
```

Comment:
1. Redundant braces
2. Return value changed, please update the docstring.

But this kinda looks wrong to change the return value here, I mean `argument` was given before to the function, right? No need to return it again.